Gemfile

```diff
@@ -1,9 +1,5 @@
 source 'https://rubygems.org'
 
-gem 'rails', '~> 7.2.0'
-
-gem 'nokogiri', '>= 1.7.1'
-
```

Comment:
I put the gems in alphabetical order. Didn't think we needed to special-case Rails and nokogiri.